pkg/api/blocks/v1.go

```diff
@@ -51,19 +56,32 @@ func (bapi *BlocksAPI) Register(r *route.Router, tracer opentracing.Tracer, logg
 }
 
 func (bapi *BlocksAPI) blocks(r *http.Request) (interface{}, []error, *api.ApiError) {
-	return bapi.blocksInfo, nil, nil
+	viewParam := r.URL.Query().Get("view")
+	if viewParam == "loaded" {
+		return bapi.loadedBlocksInfo, nil, nil
+	}
+	return bapi.globalBlocksInfo, nil, nil
```

Comment:
This feels weird from perspective on components that does not have global :thinking: 

Also I think it's on purpose that in code we talk about `global` view and in React / UI we have `blocks`?

Reply:
I agree it is weird. But in our last 1:1, me, @squat, and @GiedriusS discussed this and decided to go with this. As we have the same API but different needs in different components (Bucket viewer: a global view of blocks, Compactor: both global and loaded and Store: loaded only) we came up with two solutions:

- Modify the API logic so that we can register a single type of API multiple times in a single component with different prefixes. This means that in the Compactor, we can create two instances of Blocks API on two different prefixes (let's say `/api/v1/loaded` and `/api/v1/global`) much like how we handled the old UI.
- Extend the blocks API to have two types of data (for Global and Loaded) always and add a query parameter in the existing `/api/v1/blocks` endpoint.

The first solution is more generic but it requires us to modify logic for all of the API packages. The second solution was simpler to implement and only affected the Blocks API.